tools/crd-generator/crd-generator.go

```diff
@@ -69,19 +69,19 @@ func generateOfflineVirtualMachineCrd() {
 func generatePresetCrd() {
 	crd := generateBlankCrd()
 
-	crd.ObjectMeta.Name = "virtualmachinepresets." + v1.VirtualMachinePresetGroupVersionKind.Group
+	crd.ObjectMeta.Name = "virtualmachineinstancepresets." + v1.VirtualMachineInstancePresetGroupVersionKind.Group
 	crd.Spec = extensionsv1.CustomResourceDefinitionSpec{
-		Group:   v1.VirtualMachinePresetGroupVersionKind.Group,
-		Version: v1.VirtualMachinePresetGroupVersionKind.Version,
+		Group:   v1.VirtualMachineInstancePresetGroupVersionKind.Group,
+		Version: v1.VirtualMachineInstancePresetGroupVersionKind.Version,
 		Scope:   "Namespaced",
 
 		Names: extensionsv1.CustomResourceDefinitionNames{
-			Plural:     "virtualmachinepresets",
-			Singular:   "virtualmachinepreset",
-			Kind:       v1.VirtualMachinePresetGroupVersionKind.Kind,
-			ShortNames: []string{"vmpreset", "vmpresets"},
+			Plural:     "virtualmachineinstancepresets",
+			Singular:   "virtualmachineinstancepreset",
+			Kind:       v1.VirtualMachineInstancePresetGroupVersionKind.Kind,
+			ShortNames: []string{"vmipreset", "vmipresets"},
```

Comment:
another discussion point.  we might need presets to interact with VMs instead of VMIs